src/components/books/Books.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from '../book/Book';
+
+const Books = () => {
+  const books = useSelector((state) => state.books);
+  return (
+    <div className="books">
+      {books.length > 0 ? (
+        books.map(({ id, title, catagory }) => (
+          <Book key={id} id={id} title={title} catagory={catagory} />
```

Comment:
- [ ] Kindly remove the category from here as it is undefined and causing an error or you can give it a default value.